contribute/en/Style-and-Tone.md

```diff
@@ -89,9 +89,11 @@ Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open
 
 “Person” A human connected to a server (may be on multiple channels). “People on the server”, “People who have muted themselves” (not “Musicians” or "Channels").
```

Comment:
Servers have connections.

Reply:
A connection would be a "fader" according to the guide. We wanted to use "person" because a one person may have a number of faders on the server, and "fader" should be unambiguous because that's what you see in the UI (and it;s most likely to be what you're interested in eg, muting, volume, ordering).